tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -23,6 +23,8 @@
 #include "tt_metal/impl/program/program.hpp"
 #include "tt_metal/impl/trace/trace_buffer.hpp"
 
+#define CQ_PREFETCH_CMD_BARE_MIN_SIZE tt::tt_metal::hal.get_alignment(tt::tt_metal::HalMemType::HOST)
+
```

Comment:
Similar comment about removing if possible. Any reason it was moved to the hpp?

Reply:
Please see comment above.